images/tool-call/main.go

```diff
@@ -0,0 +1,129 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"log"
+	"os"
+
+	"github.com/ajbouh/substrate/images/tool-call/tools"
+	"github.com/ajbouh/substrate/pkg/commands"
+	"github.com/ajbouh/substrate/pkg/httpframework"
+	"tractor.dev/toolkit-go/engine"
+	"tractor.dev/toolkit-go/engine/cli"
+	"tractor.dev/toolkit-go/engine/daemon"
+)
+
+func main() {
+	prefix := os.Getenv("SUBSTRATE_URL_PREFIX")
+	engine.Run(
+		Main{},
+		&httpframework.Framework{},
+		&httpframework.StripPrefix{Prefix: prefix},
+		&commands.HTTPHandler{Route: "/commands"},
+		&commands.Aggregate{},
+		commands.NewStaticSource([]commands.Entry{
```

Comment:
You can technically put the commands.Entry directly at the top level and the Aggregate will collect them.